examples/directml/stable_diffusion/stable_diffusion.py

```diff
@@ -179,7 +183,7 @@ def optimize(
         olive_run(olive_config)
 
         footprints_file_path = (
-            Path(__file__).resolve().parent / "footprints" / f"{submodel_name}_cpu-cpu_footprints.json"
+            Path(__file__).resolve().parent / "footprints" / f"{submodel_name}_gpu-dml_footprints.json"
```

Comment:
>gpu-dml_footprints.json" [](http://example.com/codeflow?start=79&length=24)

I could not verify the stable diffusion scenario. 

Does this change works for dml after the accelerator feature is in main?

Reply:
It does! I really need to write a test for these scripts soon. Optimization shouldn't really require DML EP or a GPU, but the latency evaluator _does_ require DML. I'll try to find a way to have a test that skips evaluation but otherwise produces valid models for DML EP.